pybleau/app/plotting/tests/test_plot_styles.py

```diff
@@ -127,6 +127,12 @@ def test_colored_scatter_plot_style_view(self):
 
 @skipIf(not BACKEND_AVAILABLE, "No UI backend available")
 class TestPlotStyleBehaviors(TestCase):
+    def test_default_plot_style_supports_text_x_axis(self):
+        """ Bar plot styles must support text labels along the x axis. """
+        style = BarPlotStyle()
+        self.assertTrue(style.x_axis_style.support_text_labels)
+        self.assertFalse(style.y_axis_style.support_text_labels)
```

Comment:
Is it necessary and required that the y axis not support text labels?

Reply:
Not necessary but not supported currently. What plot whould that be? Horizontal bar plot I guess...